tests/solutions_for_students/test_grid_pathfinding_heuristics.py

```diff
@@ -0,0 +1,66 @@
+from tkinter import Grid
+import numpy as np
+from problems.grid_pathfinding.grid_pathfinding import GridPathfinding
+from problems.grid_pathfinding.grid import Grid, GridCell, GridCoord
+from problems.grid_pathfinding.heuristics.diagonal_heuristic import GridDiagonalHeuristic
+from problems.grid_pathfinding.heuristics.euclidean_heuristic import GridEuclideanHeuristic
+from problems.grid_pathfinding.heuristics.manhattan_heuristic import GridManhattanHeuristic
+
+
+def assert_equal(expected, got):
+    assert expected == got, f"expected to receive: {expected}, got: {got}"
+
+
+def problem():
+    return GridPathfinding(
+        Grid(np.full((8, 8), GridCell.EMPTY)), GridCoord(0, 0), GridCoord(0, 0), 2)
```

Comment:
If you will add a `fixture` decorator above you will be able to use it as a parameter to below functions instead of calling it, which is look more fancy in my opinion ;) 

```python
@pytest.fixture
def problem():
    return ....
.....

def test_diagonal_heuristic_in_a_goal_state_returns_0(problem):
    ....
```